addons/gdUnit3/test/ui/parts/InspectorTreeMainPanelTest.gd

```diff
@@ -37,8 +37,6 @@ func setup_test_env() -> Array:
 	tsB.add_child(_TestCase.new().configure("test_ba", 10, "/foo/TestSuiteA.gd"))
 	tsB.add_child(_TestCase.new().configure("test_bb", 20, "/foo/TestSuiteA.gd"))
 	tsB.add_child(_TestCase.new().configure("test_bc", 30, "/foo/TestSuiteA.gd"))
-	tsB.add_child(_TestCase.new().configure("test_bd", 40, "/foo/TestSuiteA.gd"))
-	tsB.add_child(_TestCase.new().configure("test_be", 50, "/foo/TestSuiteA.gd"))
```

Comment:
why you removed this test cases?
A i see you changed to have a different test case count tested on `test_suite_text_shows_amount_of_cases`

Reply:
Yes, I was creating the tests and realized all test suites had 5 cases. Then I saw that these `b` cases weren't actually used in any of the other tests.